e2e/nomostest/new.go

```diff
@@ -310,6 +310,10 @@ func FreshTestEnv(t nomostesting.NTB, opts *ntopts.New) *NT {
 
 	}
 
+	if *e2e.Kcc {
```

Comment:
This acronym is both deprecated and against Go best practices for field names. If it needs to be short, use KCC, not Kcc.

Reply:
Renamed